go/obscuronode/host/host.go

```diff
@@ -142,6 +136,14 @@ func NewObscuroAggregator(
 
 		txHandler: txHandler,
 	}
+
+	var clientServer ClientServer
+	if cfg.HasRPC {
+		clientServer = NewClientServer(*cfg.RPCAddress, &host)
```

Comment:
Minor thing but is there any need for the clientServer var here? I.e. could it just be

```
	if cfg.HasRPC {
		host.clientServer = NewClientServer(*cfg.RPCAddress, &host)
	}
```

Reply:
Cool, good catch.